els/els/urls.py

```diff
@@ -0,0 +1,8 @@
+from django.contrib import admin
+from django.urls import path, include
+
+urlpatterns = [
+    path('', include('frontend.urls')),
+    path('', include('categories.urls')),
+    path('', include('accounts.urls'))
```

Comment:
Put the `api/` prefix on the root `urls.py` so we won't need to repeat it on every other child `urls.py`
```suggestion
    path('api/', include(([
      path('', include('categories.urls')),
      path('', include('accounts.urls'))
    ]))),
```